counterpartylib/lib/api.py

```diff
@@ -764,6 +779,255 @@ def handle_root(args_path):
                 # Not found
                 return flask.Response(None, 404, mimetype='application/json')
 
+        @dispatcher.add_method
+        def sendrawtransaction(tx_hex):
+            micropayments.validate.tx_signature(dispatcher, tx_hex)
+            return backend.sendrawtransaction(tx_hex)
+
+        #########################
+        # Micropayment channels #
+        #########################
+
+        @dispatcher.add_method
+        def mpc_make_deposit(asset, payer_pubkey, payee_pubkey,
+                             spend_secret_hash, expire_time, quantity):
+            """ Create deposit and setup initial payer state.
```

Comment:
(alternatively, we could add docstrings to all API methods in this file, but I'm not convinced of that path, as it's redundant with API.md and I'd rather keep things simple...unless we added the docstrings _and_ made a script that created API docs from them dynamically)